resolvconf/resolvconf.go

```diff
@@ -39,13 +39,17 @@ var lastModified struct {
 	contents []byte
 }
 
-// Get returns the contents of /etc/resolv.conf
-func Get() ([]byte, error) {
+// Get returns the contents of /etc/resolv.conf and its hash
+func Get() ([]byte, string, error) {
```

Comment:
Can we make the `[]byte` and `hash` be defined as a data type so you can define methods based on that to calculate the hash etc.